### ISSUE TITLE
Expiry field is required but may be set to null

### ISSUE THREAD

**Issue by andrewzhurov** [ucan-wg/spec#169]
According to the [spec](https://github.com/ucan-wg/spec#32-payload), Expiry field is required but may be set to `null` to convey "never expires".
Are there reasons to have the field set to `null` vs having it absent (optional)?

There are several reasons to prefer having it optional:
1. Adheres to the [JWT spec](https://www.rfc-editor.org/rfc/rfc7519#section-4.1.4)
2. Makes for a uniform way of conveying absence, as it is for `nbf` & other fields.
3. Makes for a more sound absence semantic, [this great talk](https://www.youtube.com/watch?v=YR5WdGrpoug) gives a fine mindset for it

**Comment by Gozala** [comment]
> Are there reasons to have the field set to `null` vs having it absent (optional)?

Intention had been to have an explicit signal and catch malformed UCANs that accidentally omitted this field.

**Comment by expede** [comment]
Yeah, an expiry was required (arguably it still should be, though there are hacks around that). Ergonomically, making this field required means that people have to at least think about POLA.
